com.liferay.blade.cli/src/com/liferay/blade/cli/InitCommand.java

```diff
@@ -25,21 +25,33 @@
 
 import java.io.File;
 import java.io.FileFilter;
+import java.io.FileInputStream;
 import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Properties;
 import java.util.zip.ZipFile;
 
 import org.apache.commons.io.FileUtils;
 
 /**
  * @author Gregory Amerson
+ * @author Terry Jia
  */
 public class InitCommand {
 
+	public final String PLUGINS_SDK_7_URL =
```

Comment:
I can't get the really zip file from this url, it will return a html file if I used org.apache.commons.io.FileUtils.copyURLToFile(url, desFile)